docs/src/index.md

```diff
@@ -4,14 +4,54 @@
 CurrentModule = InfrastructureSystems
 ```
 
-**The Documentation is still under construction, some sections are unfinished. The available methods have autogenerated documentation in the internal API section**
+### Installation
 
-## Overview
+The latest stable release of PowerSystems can be installed using the Julia package manager with
+
+```Julia
+] add InfrastructureSystems
+```
+
+For the current development version, "checkout" this package with
+
+```Julia
+] add InfrastructureSystems#master
+```
+
+### Overview
 
 `InfrastructureSystems.jl` is a [`Julia`](http://www.julialang.org) package that utilities for
-the packages in NREL's [SIIP Initiative](https://github.com/NREL-SIIP). This package is meant for package development. It is used primarily by
+the packages in NREL's [SIIP Initiative](https://github.com/NREL-SIIP). This package is meant
+for module development. It is used primarily by
 [PowerSystems.jl](https://github.com/NREL-SIIP/PowerSystems.jl) and
 [PowerSimulations.jl](https://github.com/NREL-SIIP/PowerSimulations.jl) but is
-written to be extensible for other kinds of applications as well within.
+written to be extensible for other kinds of infrastructure models.
 
 This document describes how to integrate it with other packages.
+
+### Usage
+
+`InfrastructureSystems.jl` does not export any method or struct by design. Please refer to
+the [Style Guide](@ref style_guide).
+
+For detailed use of `InfrastructureSystems.jl` visit the [API](@ref API_ref) section of the
+documentation
+
+`InfrastructureSystems.jl` provides several utilities for the development of packages:
```

Comment:
This text doesn't seem to fit and doesn't really add value.  You could replace it with something like `Table of Contents`.

Reply:
I made a re-arrangement